app/models/micropost.rb

```diff
@@ -0,0 +1,4 @@
+class Micropost < ApplicationRecord
+  belongs_to :user, class_name: 'User', inverse_of: :microposts
+  validates :content, length: {maximum: 140}
```

Comment:
{}の内側にspaceを追加してください！

```suggestion
  validates :content, length: { maximum: 140 }
```